src/components/job/JobList.jsx

```diff
@@ -24,13 +26,35 @@ export default function JobList() {
     return data
       ? data.map((job) => {
           return (
-            <li key={job.id}>
-              <Link to={job.id} className={job.id && job.id == jobId ? "active" : null}>
-                <div className="card card-side w-full">
-                  <figure>LOGO</figure>
+            <li className="flex flex-row justify-center w-full mt-5" key={job.id}>
+              <Link
+                to={job.id}
+                className={`${
+                  job.id && job.id == jobId ? "active" : ""
+                }  w-4/5 rounded-xl border-slate-400 border-4`}
+              >
+                <div className="card card-compact w-full">
+                  <div
+                    className={`absolute right-0 translate-y-1/2 bottom-1/2 text-xl ${
+                      auth.user.role === ROLES.CLIENT ? "" : "hidden"
+                    }`}
+                  >
+                    <StarBorderIcon style={{ fontSize: "5vh" }} />
+                  </div>
                   <div className="card-body">
-                    <p className="card-title text-xl">{job.title}</p>
-                    <p>{job.company.name}</p>
+                    <div className="card-title text-xl">{job.title}</div>
+                    <div className="flex felx-row w-11/12">
+                      <p className="w-1/2 truncate">{job.company.name}</p>
+                      <p className="w-1/2 truncate">{`${job.location}`}</p>
+                    </div>
+                    <div className="flex felx-row w-11/12">
+                      <p className="w-1/2 flex justify-start">
+                        {job.hasMedicalBenefit ? "Offers Benefits" : "No Benefits"}
+                      </p>
+                      <p className="w-1/2  flex justify-start">
+                        Min $/hr: {job.wageHourlyMin}
+                      </p>
+                    </div>
```

Comment:
replace the value in Line 52 and 55 to some default value
add a comment called `fixme: change it when the database is done`

Reply:
Updated to new database layout